index.html

```diff
@@ -263,6 +263,52 @@ <h2>LET'S CONNECT</h2>
           </div>
         </div>
       </section>
+      <!-- Contact Form here -->
+
+      <section class="contactt">
+        <div class="ttry">
+          <form
+            class="forms"
+            action="https://formspree.io/alusinejalloh800@gmail.com"
+            method="POST"
+          >
+            <h2 class="contactme-head">Contact me</h2>
+            <p class="contactme-body">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <input
+              class="form-name"
+              type="text"
+              maxlength="30"
+              placeholder="Full Name"
+              required
+            />
+            <input
+              class="form-email"
+              type="email"
+              name="email"
+              placeholder="Email"
+              required
+            />
```

Comment:
- [ ] I think it would be better to add the name attribute in this input as well 👌